### PR TITLE
[docs] Make the examples sections more consistent

### PR DESCRIPTION
**Pull request by JesterOrNot**
Make an effort to make languages and frameworks more consistent

### CODE REVIEW

**Comment by JesterOrNot** [comment]
Sounds good

**Comment by JesterOrNot** [comment]
Fixed!

File: src/docs/bash-in-gitpod.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 Bash is the default shell for Gitpod. However, if you're developing a Bash script yourself, you may want to configure additional language support.
 
-## Bash Examples
+## Example Repositories
+
+Here are a few bash example projects that are already automated with Gitpod:
```
Comment (jankeromnes) [inline src/docs/bash-in-gitpod.md]:
```suggestion
Here are a few Bash example projects that are already automated with Gitpod:
```

**Comment by JesterOrNot** [comment]
Thanks, merging! 🚀

**Pull request merged**